src/PrestaShopBundle/Resources/views/Admin/Multistore/header.html.twig

```diff
@@ -53,11 +53,7 @@
 
     {% if isShopContext %}
       <div class="header-multishop-right">
-        {% if currentContext.hasMainUrl() %}
-          <a class="header-multishop-view-action" href="{{ link.getBaseLink(currentContext.id) }}" target="_blank" rel="nofollow">{{ "View my shop"|trans({}, 'Admin.Navigation.Header') }} <i class="material-icons">visibility</i></a>
-        {% else %}
-          <a class="header-multishop-view-action" href="{{ getAdminLink('AdminShop', true, {'shop_id' : currentContext.id}) }}" rel="nofollow">{{ "Configure URL"|trans({}, 'Admin.Actions') }} <i class="material-icons">visibility</i></a>
-        {% endif %}
+        <a class="header-multishop-view-action" href="{{ link.getBaseLink(currentContext.id) }}" target="_blank" rel="nofollow">{{ "View my shop"|trans({}, 'Admin.Navigation.Header') }} <i class="material-icons">visibility</i></a>
```

Comment:
So it's not possible anymore to configure URL @matthieu-rolland  ?

Reply:
yes, it's still possible, why?

Reply:
This PR disables the context switching link if the shop has no URL, but there is still the link to configure the URL.

Reply:
Ah I understand why you ask this: The link I remove here is the configure URL link at the top right of the header, for the current context. But since we can't switch context on a shop not having a URL specified, the configure URL will never appear at the top right of the header.

But it will still appear inside the header dropdown (see line 103)